### ISSUE TITLE
Does it work with w03 cards?

### ISSUE THREAD

**Issue by njbuch** [FlashAirDevelopers/FlashAirFileManager#1]

**Comment by tks-abe** [comment]
Hi @njbuch
Yes, it works with W03.

**Comment by njbuch** [comment]
Are you sure about that, I have problems. It seems that the card never connects to iothub with the new firmware and lua scripts, is it tested? And in that case, does it require special tricks to make it work?

**Comment by tks-abe** [comment]
Hi @njbuch
I am sorry to keep you waiting.
Although it took time to investigate the cause, it was clarified that the Lua script had some problems and may not work on W-03. I posted the modified Lua script, can you please replace it with a modified version and try it again?

- https://iot-hub.flashair-developers.com/en/static/flashair-scripts.zip
- https://github.com/FlashAirDevelopers/FlashAirFileManager/releases/tag/v0.3.2

Thanks.

**Comment by njbuch** [comment]
Nope, it doesnt work. It connects fine now with the iothub, and its niceline reporting numbers. I can browser the contents of the card using its hostname. But the FlashAirFilemanager just starts and writes "Loading..." nothing else happens.
